worlds/alttp/ItemPool.py

```diff
@@ -395,11 +395,11 @@ def generate_itempool(world):
     # rather than making all hearts/heart pieces progression items (which slows down generation considerably)
     # We mark one random heart container as an advancement item (or 4 heart pieces in expert mode)
     if world.goal[player] != 'icerodhunt' and world.difficulty[player] in ['easy', 'normal', 'hard'] and not (world.custom and world.customitemarray[30] == 0):
-        next(item for item in items if item.name == 'Boss Heart Container').advancement = True
+        next(item for item in items if item.name == 'Boss Heart Container').classification = ItemClassification.progression
```

Comment:
Since you're touching this anyways, I'm just gonna ask. Why not just mark the Sanctuary Heart Container as advancement in items.py instead of doing this, since there's only one